src/app/profile/core.nim

```diff
@@ -41,14 +41,17 @@ proc init*(self: ProfileController, account: Account) =
   let appearance = self.status.settings.getSetting[:int](Setting.Appearance)
   let messagesFromContactsOnly = self.status.settings.getSetting[:bool](Setting.MessagesFromContactsOnly)
   let sendUserStatus = self.status.settings.getSetting[:bool](Setting.SendUserStatus)
-  let currentUserStatus = self.status.settings.getSetting[:JsonNode](Setting.CurrentUserStatus)["statusType"].getInt()
+  let currentUserStatus = self.status.settings.getSetting[:JsonNode](Setting.CurrentUserStatus)
+  var statusType = 0
+  if currentUserStatus != nil:
+    statusType = currentUserStatus["statusType"].getInt()
```

Comment:
It should be possible to replace this for:
```suggestion
  let currentUserStatus = self.status.settings.getSetting[:JsonNode](Setting.CurrentUserStatus){"statusType"}.getInt()
```
since this seems to be an optional value.